migrate/migrations/20190807210857-set-default-system-limits.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+INSERT INTO config_limits (id, max)
+    VALUES 
+        ('notification_rules_per_user', 15),
+        ('contact_methods_per_user', 10),
+        ('ep_steps_per_policy', 10),
+        ('ep_actions_per_step', 20),
+        ('participants_per_rotation', 50),
+        ('rules_per_schedule', 30),
+        ('integration_keys_per_service', 30),
+        ('unacked_alerts_per_service', 200),
+        ('targets_per_schedule', 10)
+ON CONFLICT DO NOTHING;
```

Comment:
Missing default values for `heartbeat_monitors_per_service` and `user_overrides_per_schedule`.

I'd say 30 for heartbeats (match integration keys) and for overrides may be a little higher at 35. (note the limit only affects current and future overrides). They can always be adjusted manually later (in DB or UI once that launches).